### PR TITLE
RD-5956 Use new `cloudify-ui-common-*` packages

### PR DESCRIPTION
**Pull request by qooban**
## Description
This PR removes `cloudify-ui-common` package from the project dependencies and installs `cloudify-ui-common-frontend`, `cloudify-ui-common-configs` and `cloudify-ui-common-scripts` packages instead.

## Screenshots / Videos
N/A

## Checklist
- [x] My code follows the [UI Code Style](https://cloudifysource.atlassian.net/l/c/x8fq902p).
- [x] I followed the [UI Testing Guidelines](https://cloudifysource.atlassian.net/l/cp/udCVfvrx) and verified that all tests/checks have passed.
- [x] I verified if that change requires any update in the [documentation](https://cloudifysource.atlassian.net/l/c/4XKtPocy).
- [x] I added proper labels to this PR.

## Tests
Used prerelease versions first of `cloudify-ui-common-*` packages, checked if it's working in CI and moved to release versions - CI passing.

## Documentation
N/A

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -9,3 +9,4 @@ node_modules
 storybook-static
 /typings
 *.tsbuildinfo
+cloudify-ui-components-*.tgz
```
Comment (Vorbert-Kruk) [inline .gitignore:12]: It is a residue from a development phase or do we still need it? 🤔

Comment (qooban) [reply .gitignore:12]: No, added intentionally as we don't want to have that package file (generated with `npm pack`) in repository. 
Similar solution is in `cloudify-ui-common` and `cloudify-blueprint-topology`.

Comment (Vorbert-Kruk) [reply .gitignore:12]: Oh okay, thanks for the explanation
I've been looking on the [last PR for adjusting topology](https://github.com/cloudify-cosmo/cloudify-blueprint-topology/pull/226/files) but couldn't find it 
Now I can see that it has been implemented some time ago, via another PR 😁

**Review by Vorbert-Kruk — APPROVED** [review]
Thanks for addressing comments! 🎉

**Pull request merged**